### PR TITLE
feat: Activation panel 3000

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Adds the activation onboarding as a sidebar panel

Needs https://github.com/PostHog/posthog/pull/18419

## TODO
- [ ] Only show it if there is something to be done (kea throws a lot of errors when I try to connect the logics...)

## Changes

* Adds it to the sidebar
* Nothing changed in terms of the triggers
* Adds LemonProgressCircle to help replace Antd circles

Screenshot 2023-11-07 at 17 07 56

Screenshot 2023-11-07 at 17 08 03

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Review by raquelmsmith — APPROVED** [review]

File: frontend/src/layout/navigation-3000/sidepanel/sidePanelLogic.tsx [hunk 1]
```diff
@@ -58,6 +59,8 @@ export const sidePanelLogic = kea<sidePanelLogicType>([
                     tabs.push(SidePanelTab.Docs)
                 }
 
+                tabs.push(SidePanelTab.Activation)
```
Comment (daibhin) [inline frontend/src/layout/navigation-3000/sidepanel/sidePanelLogic.tsx:62]: Should we be conditionally rendering this only if they have not completed all the steps?

**Review by daibhin — APPROVED** [review]

File: frontend/src/layout/navigation-3000/sidepanel/sidePanelLogic.tsx (hunk 1 above)
Comment (benjackwhite) [reply frontend/src/layout/navigation-3000/sidepanel/sidePanelLogic.tsx:62]: See the TODO list. I tried it but kea really exploded as soon as I connected the logic to make this determination...

Comment (daibhin) [reply frontend/src/layout/navigation-3000/sidepanel/sidePanelLogic.tsx:62]: Sorry, should read the descriptions first 🙈

**Comment by benjackwhite** [comment]
Will follow up with PR to fix the activation always showing

**Pull request merged**